openedx/core/djangoapps/credential_criteria/models.py

```diff
@@ -0,0 +1,270 @@
+"""
+Models for credentials criteria.
+"""
+
+import itertools
+import logging
+
+from django.contrib.auth.models import User
+from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.functional import cached_property
+from django.utils.translation import ugettext_lazy as _
+
+from django_extensions.db.models import TimeStampedModel
+from opaque_keys import InvalidKeyError
+
+try:
+    from opaque_keys.edx.keys import LearningContextKey
+    from opaque_keys.edx.django.models import LearningContextKeyField
+    model_keyfield_type = LearningContextKeyField
+    model_key_class = LearningContextKey
+except ImportError:
+    from opaque_keys.edx.keys import UsageKey
+    from opaque_keys.edx.django.models import UsageKeyField
+    model_keyfield_type = UsageKeyField
+    model_key_class = UsageKey
+
+from . import constants, criterion_types, exceptions
+
+
+logger = logging.getLogger(__name__)
+
+
+def _choices(*values):
+    """
+    Helper for use with model field 'choices'.
+    """
+    return [(value,) * 2 for value in values]
```

Comment:
Cosmetic change for readability.

```suggestion
    return [[value, value] for value in values]
```